data/APT/data.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Aptos",
+  "name": "Aptos Coin",
```

Comment:
Is this correct? i thought this was just Aptos now

Reply:
I'll have to add an override. The top level name just needs to match the name of the L1 token. The override will allow us to display as "Aptos" though